django/db/models/sql/query.py

```diff
@@ -420,7 +414,7 @@ def get_aggregation(self, using, added_aggregate_names):
                 # clearing the select clause can alter results if distinct is
                 # used.
                 if inner_query.default_cols and has_existing_annotations:
-                    inner_query.group_by = [self.model._meta.pk.get_col(inner_query.get_initial_alias())]
+                    inner_query.group_by = tuple([self.model._meta.pk.get_col(inner_query.get_initial_alias())])
```

Comment:
Unnecessary list comprehension,  `tuple(self.model._meta.pk.get_col(inner_query.get_initial_alias()))` should do.